*** cugraph/cpp/src/utilities/cusparse_helper.cu-dist	2020-07-17 00:52:13.222701277 +0900
--- cugraph/cpp/src/utilities/cusparse_helper.cu	2020-07-17 00:52:21.429312139 +0900
***************
*** 31,38 ****
    CHECK_CUSPARSE(cusparseCreateMatDescr(&descrA));
    CHECK_CUSPARSE(cusparseSetMatIndexBase(descrA, CUSPARSE_INDEX_BASE_ZERO));
    CHECK_CUSPARSE(cusparseSetMatType(descrA, CUSPARSE_MATRIX_TYPE_GENERAL));
!   // alg = CUSPARSE_ALG_MERGE_PATH;
!   alg    = CUSPARSE_ALG_NAIVE;
    stream = nullptr;
  }
  
--- 31,38 ----
    CHECK_CUSPARSE(cusparseCreateMatDescr(&descrA));
    CHECK_CUSPARSE(cusparseSetMatIndexBase(descrA, CUSPARSE_INDEX_BASE_ZERO));
    CHECK_CUSPARSE(cusparseSetMatType(descrA, CUSPARSE_MATRIX_TYPE_GENERAL));
!   alg = CUSPARSE_ALG_MERGE_PATH;
!   // alg    = CUSPARSE_ALG_NAIVE;
    stream = nullptr;
  }
  
