*** rmm/python/setup.py-dist	2020-08-21 23:22:00.108339149 +0900
--- rmm/python/setup.py	2020-08-21 23:25:05.446798224 +0900
***************
*** 85,91 ****
      cuda_include_dir,
  ]
  
! library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
  
  # lib:
  extensions = cythonize(
--- 85,91 ----
      cuda_include_dir,
  ]
  
! library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib"), cuda_lib_dir]
  
  # lib:
  extensions = cythonize(
***************
*** 123,129 ****
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "rmm"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
--- 123,129 ----
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "rmm", "fmt", "cudart"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
