*** rmm/python/setup.py-dist	Sun Oct  4 08:20:51 2020
--- rmm/python/setup.py	Sun Oct  4 08:23:09 2020
***************
*** 57,63 ****
  
  cwd = os.getcwd()
  preprocess_files = ["gpu.pxd"]
! supported_cuda_versions = {"10.1", "10.2", "11.0"}
  
  for file_p in preprocess_files:
      pxi_file = ".".join(file_p.split(".")[:-1])
--- 57,63 ----
  
  cwd = os.getcwd()
  preprocess_files = ["gpu.pxd"]
! supported_cuda_versions = {"10.1", "10.2", "11.0", "11.1"}
  
  for file_p in preprocess_files:
      pxi_file = ".".join(file_p.split(".")[:-1])
***************
*** 85,91 ****
      cuda_include_dir,
  ]
  
! library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
  
  # lib:
  extensions = cythonize(
--- 85,91 ----
      cuda_include_dir,
  ]
  
! library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib"), cuda_lib_dir]
  
  # lib:
  extensions = cythonize(
***************
*** 99,105 ****
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "cudart"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
--- 99,105 ----
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "cudart", "fmt"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
***************
*** 123,129 ****
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "cudart"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
--- 123,129 ----
                  cuda_lib_dir,
                  os.path.join(os.sys.prefix, "lib"),
              ],
!             libraries=["cuda", "cudart", "fmt"],
              language="c++",
              extra_compile_args=["-std=c++14"],
          )
