*** cuml/cpp/src_prims/selection/knn.cuh-dist	2020-08-12 22:15:27.623075340 +0900
--- cuml/cpp/src_prims/selection/knn.cuh	2020-08-12 22:17:33.495671813 +0900
***************
*** 22,34 ****
  #include <distance/distance.cuh>
  #include <label/classlabels.cuh>
  
! #include <faiss/gpu/GpuDistance.h>
! #include <faiss/gpu/GpuIndexFlat.h>
! #include <faiss/gpu/GpuResources.h>
! #include <faiss/gpu/StandardGpuResources.h>
! #include <faiss/utils/Heap.h>
! #include <faiss/gpu/utils/Limits.cuh>
! #include <faiss/gpu/utils/Select.cuh>
  
  #include <thrust/device_vector.h>
  #include <thrust/iterator/transform_iterator.h>
--- 22,34 ----
  #include <distance/distance.cuh>
  #include <label/classlabels.cuh>
  
! #include <faiss/faiss/gpu/GpuDistance.h>
! #include <faiss/faiss/gpu/GpuIndexFlat.h>
! #include <faiss/faiss/gpu/GpuResources.h>
! #include <faiss/faiss/gpu/StandardGpuResources.h>
! #include <faiss/faiss/utils/Heap.h>
! #include <faiss/faiss/gpu/utils/Limits.cuh>
! #include <faiss/faiss/gpu/utils/Select.cuh>
  
  #include <thrust/device_vector.h>
  #include <thrust/iterator/transform_iterator.h>
***************
*** 289,295 ****
        select_stream(userStream, internalStreams, n_int_streams, i);
  
      gpu_res.noTempMemory();
!     gpu_res.setCudaMallocWarning(false);
      gpu_res.setDefaultStream(device, stream);
  
      faiss::gpu::GpuDistanceParams args;
--- 289,295 ----
        select_stream(userStream, internalStreams, n_int_streams, i);
  
      gpu_res.noTempMemory();
!     // gpu_res.setCudaMallocWarning(false);
      gpu_res.setDefaultStream(device, stream);
  
      faiss::gpu::GpuDistanceParams args;
