*** cuml/python/setup.py-dist	2020-09-16 07:51:50.124443584 +0900
--- cuml/python/setup.py	2020-09-16 07:53:43.634115688 +0900
***************
*** 141,147 ****
  
      def initialize_options(self):
  
!         self.singlegpu = False
          super().initialize_options()
  
      def finalize_options(self):
--- 141,147 ----
  
      def initialize_options(self):
  
!         self.singlegpu = True
          super().initialize_options()
  
      def finalize_options(self):
***************
*** 151,157 ****
          # object has all the args used by the user, we can check that.
          self.singlegpu = '--singlegpu' in self.distribution.script_args
  
!         libs = ['cuda', 'cuml++', 'rmm']
  
          include_dirs = [
              '../cpp/src',
--- 151,157 ----
          # object has all the args used by the user, we can check that.
          self.singlegpu = '--singlegpu' in self.distribution.script_args
  
!         libs = ['cuda', 'cuml++', 'rmm', 'cudart', 'fmt']
  
          include_dirs = [
              '../cpp/src',
***************
*** 166,172 ****
          python_exc_list = []
  
          if (self.singlegpu):
!             python_exc_list = ["*.dask", "*.dask.*"]
          else:
              libs.append('cumlprims')
              libs.append('nccl')
--- 166,172 ----
          python_exc_list = []
  
          if (self.singlegpu):
!             python_exc_list = ["*.dask", "*.dask.*",  "*.*.dask", "*.*.dask.*", "*.dask.*.*","*.*.dask.*.*"]
          else:
              libs.append('cumlprims')
              libs.append('nccl')
***************
*** 208,214 ****
  
      def initialize_options(self):
  
!         self.singlegpu = None
  
          super().initialize_options()
  
--- 208,214 ----
  
      def initialize_options(self):
  
!         self.singlegpu = True
  
          super().initialize_options()
  
